src/commands/build.ts

```diff
@@ -110,6 +110,11 @@ export class BuildCommand implements Command {
       type: String,
       description: 'Updates the <base> tag if found in the entrypoint document.'
     },
+    {
+      name: 'npm',
+      type: Boolean,
+      description: 'Look for dependencies in "node_modules/".'
```

Comment:
Not really sure if this is the right way to put this. `--npm` for `build` just sets `componentDir: "node_modules/"` but the idea is that if there were other configurable things that end up in polymer.json that are npm related, then it should set those too. Also, there's an implicit `--component-dir` flag, given that change to polymer-project-config. Maybe I should just document both of these for all the commands.